### PR TITLE
Dg usecase userstory

### PR DESCRIPTION
**Pull request by Gabriel4357**
Updated Use Case and User stories

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1819,27 +1688,27 @@ otherwise)
...
-    - 1b1. DMS informs the logged-in owner of invalid delivery id being entered.
+    - 1b1. DMS informs the logged-in owner of Invalid delivery id being entered.
 
       Use case ends.
 
-- 1c. Logged-in owner entered date in the wrong format.
+- 1c. Logged-in owner did not specifiy a Delivery ID.
 
-    - 1c1. DMS informs the logged-in owner of invalid format and shows the correct format.
+    - 1c1. DMS informs the logged-in Owner to specify a Delivery to update.
```
Comment (zhonghan721) [inline docs/DeveloperGuide.md:1705]: Perhaps it will be good to talk about the invalid date format/invalid date.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1318,21 +1318,21 @@ otherwise)
 
 **Extensions:**
```
Comment (zhonghan721) [inline docs/DeveloperGuide.md:1319]: Perhaps can add in the case for duplicated fields/prefixes

**Review by zhonghan721 — COMMENTED** [review]
Some general comments

**Review by jianyangg — APPROVED** [review]
LGTM

**Pull request merged**